include/mbgl/style/expression/compound_expression.hpp

```diff
@@ -141,6 +141,18 @@ ParseResult createCompoundExpression(const CompoundExpressionRegistry::Definitio
 ParseResult createCompoundExpression(const std::string& name,
                                      std::vector<std::unique_ptr<Expression>> args,
                                      ParsingContext& ctx);
+    
+ParseResult createCompoundExpression(const std::string& name, ParsingContext& ctx);
```

Comment:
Could we stick a quick comment here indicating that these overloads are here to avoid inconvenience of having to create the `args` vector before calling?